src/client.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, address, nickname=''):
         if nickname:
             self.changeNick(nickname)
         else:
-            self.changeNick(raw_input('Nickname: '))
+            self.changeNick(raw_input('What\'s Nickname You want to be shown: '))
```

Comment:
Looks good! I think the following is slightly better grammar? What do you think?

```suggestion
            self.changeNick(raw_input('What nickname do you want to be shown: '))
```

Reply:
Sure It is great!!